src/test/ui/rfc-2091-track-caller/caller-location-intrinsic.rs

```diff
@@ -1,21 +1,27 @@
 // run-pass
 
-#![feature(core_intrinsics)]
+#![feature(track_caller)]
+
+#[inline(never)]
+#[track_caller]
+fn defeat_const_prop() -> &'static core::panic::Location<'static> {
```

Comment:
Would be good to have a comment why/how this "defeats" const_prop. I have no idea what is going on here.

Reply:
Good point, I think the function name is clearer in https://github.com/rust-lang/rust/pull/67137/commits/507cb07f1684f116f4e4cf08435fb50ae501987c. Feel free to chime in review there!